README.md

```diff
@@ -1,41 +1 @@
-# Gas-station-tracker
```

Comment:
Please make sure the proposal aren't deleted. This proposal is currently deleted in the pull request